presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -243,4 +261,28 @@ private MaterializedResult getExpectedOrdersTableDescription(boolean dateSupport
                     .build();
         }
     }
+
+    @Test
+    public void testCreateSchema()
+            throws SQLException
+    {
+        skipTestUnless(canCreateSchema());
```

Comment:
make sure that schema does not exist before creation, that it doesn't exists after deletion